core/src/test/scala/opal/ImmutabilityLattice.scala

```diff
@@ -31,17 +31,31 @@ case object Immutable extends Immutability
 object Immutability {
 
   implicit object ImmutabilityLattice extends Lattice[Immutability] {
-    override def join(current: Immutability, next: Immutability): Immutability = {
-      if (<=(next, current)) current
-      else next
+    override def join(v1: Immutability, v2: Immutability): Immutability = {
+      if (<=(v2, v1)) v1
```

Comment:
Should use infix syntax.

Reply:
I have renamed this to lteq to match the naming of PartialOrderingWithBottom.